tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/dashboard.html

```diff
@@ -1,185 +1,289 @@
+<div class="popupWait hide"></div>
+<div class="loadingImage hide" id="popupWaitLoadingImage">
+    <a title="Matter Center Home" id="mcIcon" href="/sitepages/MatterCenterHome.aspx" target="_parent">
+        <img title="Matter Center" src="../Images/WindowsLoadingFast.GIF" alt="Matter Center" />
+    </a>
+</div>
+<div class="popupContainerBackground hide"></div>
+<div id="mainDivContainer">
+    <div class="matterPopup"></div>
+    <div class="documentPopup"></div>
+    <div class="clear"></div>
+    <!--Start Pop up conatins for documents to send documents to one drive, send email etc-->
+    <div class="popUpContainer hide">
+        <header class="popUpHeader">
+            <div class="popUpHeaderTitle popUpFloatLeft"></div>
+            <div class="notificationContainerForPopup"></div>
+            <img class="mailCartCloseIcon" alt="Close" src="../Images/close-666.png" title="Close" />
+        </header>
+        <div id="PopupBody">
+            <div class="selectMailFormat">
+                <div class="sendEmailOptions sendToOneDriveOption" onclick="sendToBriefcase();">
+                    <img class="mailCartLogo" src="../Images/my-briefcase-666.png" alt="Send to My OneDrive for Business" title="Send to My OneDrive" />
+                    <div title="Send to My OneDrive for Business" class="inlineValues">Send to My OneDrive for Business</div>
+                </div>
+                <div class="sendEmailOptions emailAttachmentOption" onclick="downloadEmail(true);">
+                    <img class="mailCartLogo" src="../Images/attach_666_27X27.png" alt="Email as attachment" title="Email as attachment" /><div title="Email as attachment" class="inlineValues">Email as attachment</div>
+                </div>
+                <div class="sendEmailOptions emailLinkOption" onclick="downloadEmail(false);">
+                    <img class="mailCartLogo" src="../Images/link-666.png" alt="Email as link" title="Email as link" /><div title="Email as link" class="inlineValues">Email as link</div>
+                </div>
+            </div>
+            <div class="popUpBody popUpFloatLeft">
+                <div class="checkOutPopup">
+                    <div class="checkOutPopupPointer"></div>
+                    <div class="checkOutPopupData">
+                        <div class="checkOutMetadata marginTop10"></div>
+                    </div>
+                </div>
+                <ul class="emailDocumentNames"></ul>
+                <div class='pinLoadingImage' id='mailCartLoadingImage'>
+                    <img src='/Images/WindowsLoadingFast.GIF' title='Loading' class='LoadingImage' alt='Loading...' />
+                </div>
+            </div>
+        </div>
+    </div>
+    <!--End Pop up conatins for documents to send documents to one drive, send email etc-->
+    <div class="matterCenterHeaderBackground hide"></div>
+    <!--Main dashboard content-->
+    <div id="contentDiv">
+        <div class="row searchdashboard">

```

Comment:
don't mix bootsrap class with other classes. bettor change this to <div class="row"> <div class=searchdashboard>